tests/e2e/test_rosacli_operator_roles.go

```diff
@@ -90,6 +90,7 @@ var _ = Describe("Edit operator roles", labels.Feature.OperatorRoles, func() {
 				By("Delete in-used oidc config in auto mode")
 				output, err = ocmResourceService.DeleteOIDCConfig(
 					"--oidc-config-id", oidcConfigID,
+					"--region", "us-west-2",
```

Comment:
why hardcoding the region ? Shouldn't we take it from the cluster profile/details instead ?

Reply:
fixed